config/clusterPipeline.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sinks":
```

Comment:
Yep, that's the plan for now. I did verify that this will come up safely, though.